Sources/NavigationStack/NavigationStack.swift

```diff
@@ -56,6 +71,8 @@ public class NavigationStack: ObservableObject {
         }
     }
 
+    /// Navigates back to a previous view.
+    /// - Parameter to: The destination type of the transition operation.
```

Comment:
```suggestion
    /// - Parameter to: The destination type of the navigation operation.
```
?

Reply:
I've always referred to "transition" when talking about the "push/pop actual operation", so I'd prefer leave it as "transition".